### PR TITLE
Adds casatunes, alko and wasteplan_trv integrations.

### PR DESCRIPTION
**Pull request by jonkristian**

### CODE REVIEW

File: components/wasteplan_trv.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "wasteplan_trv",
+    "owner": [
+        "@jonkristian"
+    ],
+    "manifest": "https://raw.githubusercontent.com/jonkristian/wasteplan_trv/master/custom_components/wasteplan_trv/manifest.json",
```
Comment (ludeeus) [inline components/wasteplan_trv.json:6]: This integration does not have any requirements to build wheels for, so it should not be added here.

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Comment by ludeeus** [comment]
@frenck my comment was for 1 of the 3 files added in this PR :+1:

**Review by frenck — CHANGES_REQUESTED** [review]
Oh jeez, sorry @ludeeus didn't notice that.

@jonkristian Please don't add multiple concerns in a single PR. Split this PR in separate ones.

**Comment by jonkristian** [comment]
Sorry about this guys. I'm not too savvy with PRs and how to split a PR, can I just close this PR and create new ones?

**Comment by frenck** [comment]
>  I'm not too savvy with PRs and how to split a PR, can I just close this PR and create new ones?

Yeah sure, that will work.

**Pull request closed without merging**